Updating for MCP4728A4

BlitzCityDIY opened the pull request:
Updating library to be able to pass 0x64 I2C address for MCP4728A4 variant (see revision history on product copy: https://www.adafruit.com/product/4470).

BlitzCityDIY committed:
Updating for MCP4728A4

Updating library to be able to pass 0x64 I2C address for MCP4728A4 variant

BlitzCityDIY committed:
Updating for CI

Updating for CI

tekktrik commented:
Did the CI add all those extra lines in the example file?  If possible, it may be more legible to keep it as it was.

For the library, thanks for the type annotation!  Perhaps it might be nice to add the different I2C addresses as public global variables at the top:

```python
MCP4728_DEFAULT_ADDRESS = 0x60
MCP4728A4_DEFAULT_ADDRESS = 0x64
```

Then we can additionally encourage people to use them as/if needed:

```python
i2c = board.I2C()
mcp4728 = MCP4728(i2c, address=MCP4728A4_DEFAULT_ADDRESS)
```

What do you think, would that be helpful?

BlitzCityDIY commented:
Thanks! It totally did add all those lines so I just removed those. This is my first time contributing to a library so I appreciate the feedback! I think global variables would definitely be helpful and I could point them out in the Learn guide. You're saying these variables would be in the example files?

tekktrik commented:
If you change `_MCP4728_DEFAULT_ADDRESS` to `MCP4728_DEFAULT_ADDRESS` up at the top and add `MCP4728A4_DEFAULT_ADDRESS = 0x64` underneath, you could use them in an example file like such:

```python
import board
import adafruit_mcp4728

i2c = board.I2C()  # uses board.SCL and board.SDA

# Use the default address for the MCP4728A4 variant of the board
mcp4728 = adafruit_mcp4728.MCP4728(i2c, address=adafruit_mcp4728.MCP4728A4_DEFAULT_ADDRESS)

...
```

Just make those changes and push the fix for the extra spaces and it should be good to go!

BlitzCityDIY committed:
Updating I2C addresses

Updating both variants of the board as public global variables and adding an example in the main example file. Also fixing spacing from original local pre-commit run

BlitzCityDIY commented:
got it, yeah i definitely like that. just PR'd that update

tekktrik reviewed (approved):
Looks great to me, nice!
